pom.xml

```diff
@@ -196,6 +205,8 @@
             </activation>
             <properties>
               <test.compilation.export.target>ALL-UNNAMED</test.compilation.export.target>
+              <skip.native.compile>true</skip.native.compile>
+              <build.native.file>placeholder_does_not_exist</build.native.file>
```

Comment:
I think `placeholder_does_not_exist` is a bit confusing. Maybe something like `None_not_really_used_here`, or anything else that is easier to get?

Reply:
Updated this value to be `None_not_really_used_here` which also seems reasonable and does convey the message that the value is not really used.